kubectl-plugin/src/main.rs

```diff
@@ -18,12 +23,15 @@ use yaml_rust::YamlLoader;
 
 #[derive(StructOpt, Debug)]
 struct CliArgs {
-    /// Rest endpoint.
+    /// The rest endpoint, parsed from KUBECONFIG, if left empty .
     #[structopt(long, short)]
     rest: Option<Url>,
     /// The operation to be performed.
     #[structopt(subcommand)]
     operations: Operations,
+    /// The Output, viz yaml, json.
```

Comment:
what is viz?

Reply:
viz, is acronym for namely. For example kind of.

Reply:
had not come across that, thanks :+1: